src/utils/supabase.ts

```diff
@@ -841,3 +845,152 @@ export const getChannel = async (
     message: "Successfully fetched the channel!",
   };
 };
+
+export const getTutors = async (
+  queryData: TutorQuery
+): Promise<SupabaseResponse<TutorSelect[]>> => {
+  const query = supabase.from("tutors").select("*");
+
+  addQueryFilters(query, queryData);
+
+  const tutorsResponse = await query;
```

Comment:
**suggestion:** This can be combined with the query on line 852

Reply:
I will combine the lines if `addQueryFilters` is removed, for readability